Coalition Games (#4905)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			`	As you turn to look, one of the participants' "chariot" hits one of the trees, and they slow down, eventually coming to a full stop. Although the drones following each Saryd isn't transmitting the sound (the only sounds being voices of who you believe are commentators, though you can't understand their language), it is clear from his overall expression and the way he threw the readied arrow at the ground makes you realize he has probably been disqualified.`
			`	While they move to a more convoluted part of the woods, with several tall brambles and many ledges, the targets start appearing at a progressively faster rate. One of the monitors focuses on one particular racer, who manages to not miss a single shot in a segment with seven different targets in a row. Looking at a smaller, but still massive screen with pictures of the contestants, you see that the picture of this racer rises many positions. It seems that hitting these targets count for gaining points during the race.`

> 			`	You couldn't count how many there were at the start of the race, due to the sheer number of Saryds and the way they rapidly changed course multiple times during the competition, but as the race approaches it's end, and some of the contestants cross a stream serving as the finish line, you manage to identify that there are thirty that made it. The drones end their transmission, as the monitors cut to a group on the ground that starts interviewing the ones that made it.`

Author: keeping Saryds

			`	You go with the crowd, and are shown to a room with many elevators. Eventually, everyone descends to the valley below, and follow a path to an actual, full-size stadium, and a massive one at that.`
			`	Awkwardly sitting in the middle of all these alien beings, it hits you that you're probably the first human ever to witness these games. You eye the stadium grounds, oddly it has some small mounds, and six holes: four in the corners, and two across the middle of the longer side of the rectangular grounds.`
			`	Roughly half an hour passes, and a Kimek sitting next to you, who thankfully had a translation device, explains that the contestants are resting. A couple more minutes pass, and the thirty Saryds enter the grounds, and are divided into six teams, each donning a different color, and equipped with what looks like some fancy polo mallets. You see that Ulmodi is on the blue team. Several large spheres are placed at the center of the stadium, with one member of each team around them, forming a circle.`

> 			`	The game starts, some sort of mix between polo, billiards, and golf. The teams trot around for a while, then rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over, sometimes even accidentally being flung to the audience.`

Author: is this paragraph a big issue? tried coming up with some ways to change it up but nothing really came to mind, so just to know if I should continue with trying to get something that works better

			`	Eventually one of them enters one of the holes, which lights up with the color of the team that made the shot: the red team has scored a point. Many of the crowd stand up and a multitude of screams echo across the arena, you're not sure if everyone here is a fan of the red team, or if they all will cheer whenever someone gets a point. The sphere is brought back up automatically via a tube close to the hole, and it's back in the game.`
			`	Points keep rolling in as the teams score goals, with eventually the blue team getting their fifth point. The crowd gets even crazier than before, and the entire blue team comes together and celebrates, while the other five teams aren't quite as enthusiastic. The commentators say something, and the helpful Kimek sitting beside you once again helps you understand it all.`

> 			`	"The winning score, they have reached! The winners, the blue team are!" They say, still in the cheering mood. After everyone settles down, the members of the teams are shown in the replay screen, going from the team that scored the least, all the way to the blue team at the end. Ulmodi is placed third, which must mean a lot, considering him and the other two above him celebrate by running laps around the inside of the stadium.`

Author: I didn't want to go through the process of creating an entire points system for this, or to make the textwalls even larger in explaining it, but the general ideal would be that each individual ranking as well as the team's placing would influence in the "points" one would get. the thirty that make it are divided into six teams of five, so I'd say the teams are formed based on the "points" each one has. both the individual and team pointing would be deciding factors in whether a contestant gets to compete in the cooperative match, and the individual points would be what matter when deciding whether some of those that pass said match are selected for being elevated to Heliarch rank

			`	In the starting seconds, dozens of Kimek fall from the ring, onto the nets on the side, and are helped back to the surrounding seats by several workers. It seems this first stage works like a collective sumo match. After enough have fallen out, another whistle is heard, and everyone stops fighting. Heliarch agents get back on the ring and reorganize those that remain into two teams once again. This time about a minute passes before anyone falls off, the fighters have some actual personal space to work with this time, so they're probably much more comfortable fighting this way.`
			`	This process repeats a few more times, until finally all the ones that remained after everything are gathered in the center of the ring. You estimate that about less than one percent of the ones that first entered. Everyone leaves the ring, and you're standing there in confusion for a while, unsure if the game has ended.`
			`	You feel something tap the top of your head, and turn around to see an Arach sitting behind you, "Confused, the human visitor is?" Their little box translates.`

> 			`	You nod, and they decide to briefly breakdown the competition during this break. "Numerous, the Kimek are, so more athletes, they have," they say. "Tens of thousands, at first enter. Redone, the teams are, once in half, that number is cut."`

Author: also changed to "Over a thousand" to have the math make more sense

			`	You find Pairri is set to fight close to where you're sitting, so you get a good view of her fights. While she is short even by Kimek standards, she stands her ground and manages to beat some slightly larger opponents, driving them out of the circular limit of the rink.`
			`	Above the large arena some monitors depict the fighters and seem to show their score: fighters that lose a match don't leave altogether, instead they rotate counter clockwise, fighting others to try and climb back up to the winning spots.`
			`	Eventually, after many of these rounds and rotational swap outs, Pairri and her next opponent wait as the other forty eight finish their own fights, then move to the center of the field.`
			`	"Undefeated, the both of them are," the Arach behind you says. "The floor victor, the winner of this final match will be."`

> 			`	The whistle sounds again and Pairri charges at her slightly larger opponent, pressuring them and getting them close to the edge. They redirect the force by going to their side, and both start a battle of attrition close to the limits of their rink. Her opponent repositions themselves to the middle, and tries to charge at her, but she closes the gap and they start wrestling in the middle. After a while they get back to the border, with Pairri pressing the other Kimek once again, as they get closer and closer to being out of bounds. Suddenly, her opponent manages to grip her with their frontal legs, and by lifting their own body up with the other four, just barely launching her up.`

Author: the idea is that Pairri's opponent goes back to the middle, with this change wouldn't it then change the meaning to they move Pairri to that position?

> 			`	The whistle sounds again and Pairri charges at her slightly larger opponent, pressuring them and getting them close to the edge. They redirect the force by going to their side, and both start a battle of attrition close to the limits of their rink. Her opponent repositions themselves to the middle, and tries to charge at her, but she closes the gap and they start wrestling in the middle. After a while they get back to the border, with Pairri pressing the other Kimek once again, as they get closer and closer to being out of bounds. Suddenly, her opponent manages to grip her with their frontal legs, and by lifting their own body up with the other four, just barely launching her up.`

Reviewer: I misunderstood what you meant. In this cause, you would want to switch “themselves” to “themself”.

			`	Once the 'rafts' are full, they leave their docks and start flying outwards, and as soon as you're outside, you're greeted by a lush rainforest, as you fly beside some massive trees, bird-like creatures with funneled beaks flying to and from them and the carpet of green below. In the distance, you see what appears to be an ocean, but one of the Heliarch agents in charge of the raft says that it's actually a river.`
			`	Getting closer and closer to a spot where multiple of the colossal trees are in close proximity to one another, you arrive at the same time as some slightly smaller rafts, these ones with symbols on their coverings, a different one for each. The Heliarch agents on your own raft are too busy getting the raft positioned, from the looks of it, so you ask a Saryd seated next to you, after noticing they're carrying a translation box.`
			`	"The multiple Arach houses, the symbols they represent," she says. "To be patrons of atheletes, is a custom of members of the great houses."`

> 			`	The different rafts all start slowly lowering their altitude, getting close to the center of where all the massive plants, that resemble oak trees, but with leaves many times the usual size, and many vines hanging from the branches, and even extending to other trees.`

Author: which one?

> 			`	The different rafts all start slowly lowering their altitude, getting close to the center of where all the massive plants, that resemble oak trees, but with leaves many times the usual size, and many vines hanging from the branches, and even extending to other trees.`

Reviewer: The entire line 275 is one sentence with numerous commas to add description. It should probably be split up into too sentences. Additionally, the sentence has no object. If you remove all the description within commas, the sentence reads, “ The different rafts all start slowly lowering their altitude, getting close to the center of where all the massive plants.”

> 			`	The different rafts all start slowly lowering their altitude, getting close to the center of where all the massive plants, that resemble oak trees, but with leaves many times the usual size, and many vines hanging from the branches, and even extending to other trees.`

Reviewer: The entire line 275 is one sentence with numerous commas to add description. It should probably be split up into too sentences. Additionally, the sentence has no object. If you remove all the description within commas, the sentence reads, “ The different rafts all start slowly lowering their altitude, getting close to the center of where all the massive plants.”

			`	Up above, you can finally see the entire stage: a long, mostly flat stretch of land, leading to a rocky wall, cut to form a perfect right angle with the ground. At the top, there is a river bank, with many logs laid on the ground. The river itself is not terribly big, but the current seems to be potent enough that the boats moving around the waters have some difficulty turning at times.`
			`	The twenty teams line up, with the Saryds in front of a row of chariots, and the other two of each team being on top of said chariot. Zooming in on Tumub, you see that he is helping the Kimek of their group check a bag with some tools, while he himself has one on his back.`
			`	"In many ways we differ, when it comes to physical form," Pairri speaks to you. "Different tasks, each member is assigned. For the sake of fairness, it is so."`
			choice
				`	"What are they carrying in those bags?"`
				`	"So are those chariots for the 'task' of the Saryds?"`

> 			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping towards the wall. The Kimeks and Arachs of each chariot do their best to stay on, as the alien centaurs give their all to the race.`

Author: also changed removed silly `alien centaurs` and replaced with `Saryd teammates`

			`	The Kimek reach the top relatively quickly, moving up with great speed; once at the top, they open their bags, revealing a multitude of what seem to be carpenter tools, and a lot of rope.`
			`	"To cross the great river, one of the goals of the game is," says Pairri. "For our great agility, us Kimek are known. To build the raft, our task is."`
			`	Back down where the chariots are, the Arachs have started to prepare for a climb, having a large piece of cloth attached to some elastic-like fabric, which they wear over their bodies like some climbing equipment. The Saryds nestle themselves on the cloth, and after a few short test climbs, to see if everything is stable, the Arach start climbing.`
			`	Tumub's movements appear very calculated, never having more than one leg not touching the wall at a time, carefully looking for the best spots to anchor himself on, and continue carrying Ulmodi's much larger frame up the stone barrier.`

> 			`	As soon as they finish the climb, the Saryds get up, unstrap the fabric from the Arachs, and start carrying their exausted companions toward the now nearly complete raft. The Saryds lay the Arachs beside the raft, and help the Kimeks properly secure the ropes. The Kimeks and Saryds help the Arachs get on, and start pushing the raft to the lake, with the Saryds getting onto them shortly before they touch the water, and the Kimek waiting until the last second.`

Reviewer 2: ```suggestion
			`	As soon as they finish the climb, the Saryds get up, unstrap the fabric from the Arachi, and start carrying their exhausted companions toward the now nearly complete raft. The Saryds lay the Arachi beside the raft and help the Kimeks properly secure the ropes. The Kimek and Saryds help the Arachi get on and start pushing the raft to the lake. The Saryds get onto them shortly before they touch the water, and the Kimek wait until the last second.`
```
```suggestion
			`	As soon as they finish the climb, the Saryds get up, unstrap the fabric from the Arachi, and start carrying their exhausted companions toward the now nearly complete raft. The Saryds lay the Arachi beside the raft and help the Kimeks properly secure the ropes. The Kimek and Saryds help the Arachi get on and start pushing the raft to the lake. The Saryds climb on shortly before they touch the water, but the Kimek wait until the last second.`
```

> 			`	After focusing on Ulmodi and Tumub for a while, you notice that the Kimek from their group picked up three small paddles, and is working to fight the river's flow. Ulmodi picks up something that was between some of the logs, that you figured was a part of the boat itself, but turns out to be a thin, long wooden structure, with a mix of a shovel and a pickaxe on one end. Ulmodi uses the pick side to 'hook' some rocks sticking up, and holds as much as he can before letting go, slightly tilting the boat back to the straight course they first started at.`

Reviewer 2: ```suggestion
			`	After focusing on Ulmodi and Tumub for a while, you notice that the Kimek from their group picked up three small paddles and is working to fight the river's flow. Ulmodi picks up something that was between some of the logs that you figured was a part of the boat itself but turns out to be a thin, long wooden structure with a mix of a shovel and pickaxe on one end. Ulmodi uses the pick side to 'hook' some rocks sticking up and holds as much as he can before letting go, slightly tilting the boat back to the straight course they first started at.`
```
```suggestion
			`	After focusing on Ulmodi and Tumub for a while, you notice that the Kimek from their group has picked up three small paddles and is working to fight the river's flow. Ulmodi grabs a tool that looks like a combination between a shovel and pickaxe from its spot wedged between two logs. He uses the pick side to grab onto some rocks sticking up and pulls, tilting the boat back on course.`
```

> 			`	About halfway across, Tumub gets up, having caught his breath, and heads to the sail. Standing up on two legs, he opens up the sail, and skillfully manages it using the ropes, altering the form of the sail to better adapt to the changing needs of the turbulent vessel.`

Reviewer 2: ```suggestion
			`	About halfway across, Tumub gets up, having caught his breath, and heads to the sail. Standing up on two legs, he opens up the sail and skillfully manages it using the ropes, altering the form of the sail to better adapt to the changing needs of the turbulent vessel.`
```
```suggestion
			`	About halfway across, Tumub has caught his breath and is mans the sail. Standing up on two legs, he opens it up and skillfully manages it using the ropes, altering the direction of the sail to better adapt to the changing needs of the turbulent vessel.`
```

> 			`	As you and Pairri cheer for the group, you notice that not everyone is having the same success at crossing. Two rafts have gone dangerously too far down the river, not being able to fight the stream, and the Heliarch boats head to them at full speed, rescuing the competitors before anything drastic happens.`

Reviewer 2: ```suggestion
			`	As you and Pairri cheer for the group, you notice that not everyone is having the same success crossing. Two rafts, unable to fight the stream, have drifted dangerously far down the river. Heliarch boats speed toward them, rescuing the competitors before anyone is hurt.`
```
Suggested changes (personal preference).

> 			`	In total, eighteen teams have made it across the river, with Ulmodi, Tumub and their Kimek partner winning second place by a hair, seeing as almost everyone arrived at the other margin roughly at the same time. As the competitors themselves celebrate with their own teams, and later gather all together in a large group to also applaud their opponents, the crowd is ecstatic, as praise heard in all three Coalition languages echoes all over.`

Reviewer 2: ```suggestion
			`	In total, eighteen teams make it across the river. The race was extremely close the whole time, but Ulmodi, Tumub, and their Kimek parter win second place by a hair. The competitors celebrate with their team and later gather to commend their opponents while the crowd roars. Cheering in all three Coalition languages fills the stadium.`
```
Suggested changes (personal preference).

> 			`	Everyone grows quiet, however, as three Heliarch Punishers come down for landing right in front of the athletes. Out come who you assume are the captains of each ship, Heliarchs wearing the same type of circlet you saw in the Ring of Friendship, with each of them being from a different Coalition species.`

Reviewer 2: ```suggestion
			`	Everyone grows quiet, however, as three Heliarch Punishers come down for landing right in front of the athletes. Out come who you assume are the captains of each ship: Heliarchs wearing the same type of circlet you saw in the Ring of Friendship, each of them being from a different Coalition species, emerge.`
```
```suggestion
			`	Everyone grows quiet, however, as three Heliarch Punishers land right in front of the athletes. One Heliarch, wearing the same type of circlet you saw in the Ring of Friendship, emerges from each ship, presumably their captains. There is one Arach, one Saryd, and one Kimek.`
```

> 			`	"Deemed worthy, some have been. The winners, the Heliarchs shall now announce," Pairri says. As the Heliarchs start speaking she tries to summarize it all and let the box translate it, but the device quickly becomes overwhelmed. Clearly whatever it is that they are saying must be some sort of grand finale speech, judging by the way the crowd responds during the few pauses.`

Reviewer 2: ```suggestion
			`	"Deemed worthy, some have been. The winners, the Heliarchs shall now announce," Pairri says. As the Heliarchs start speaking, she tries to summarize it all and let the box translate it, but the device quickly becomes overwhelmed. Clearly, whatever it is that they are saying must be some sort of grand finale speech, judging by the way the crowd responds during the few pauses.`
```
```suggestion
			`	"Deemed worthy, some have been. The winners, the Heliarchs shall now announce," Pairri says. As the Heliarchs start speaking, she tries to summarize and let the box translate it, but the device is unable to keep up with the rate at which she speaks. Judging by the audience's reaction during pauses, they must be giving some sort of grand finale speech.`
```

> 			`	After they finish, the monitor switches to a black screen, which after a moment lights up with live footage of one of the athletes, a Kimek. Then a Saryd, another Kimek, an Arach. In the end both Tumub's and Ulmodi's faces also show in the monitor, and Pairri does what you think is the Kimek version of an excited scream.`

Reviewer 2: ```suggestion
			`	After they finish, the monitor switches to a black screen which after a moment lights up with live footage of one of the athletes, a Kimek, then a Saryd, another Kimek, and an Arach. In the end, both Tumub's and Ulmodi's faces also show in the monitor, and Pairri does what you think is the Kimek version of an excited scream.`
```
```suggestion
			`	After they finish, the monitor switches to a black screen, which after a moment displays live footage of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, and Pairri makes a sound that you think sounds like an excited scream - or, at least, a Kimek version of such a thing.`
```

			`	"Selected, my friends have been!" She says. "Made it, they have!"`

> 			`	There's a few more words said after everyone has seen the results and settled down, and the six that were selected are invited into the Punishers. Pairri explains that they are being considered to be elevated to the rank of Heliarch, and will go to the Ring of Friendship, to continue their journey to it.`

Reviewer 2: ```suggestion
			`	There's a few more words said after everyone has seen the results and settled down, and the six that were selected are invited into the Punishers. Pairri explains that they are being considered to be elevated to the rank of Heliarch and will go to the Ring of Friendship to continue their journey toward it.`
```
```suggestion
			`	A few more words are said after the crowd has settled, and the six that were selected are invited into the Punishers. Pairri explains they are being considered for the rank of Heliarch and will go to the Ring of Friendship to continue their journey.`
```

> 			`	The ring platforms head back to the building, and as you and Pairri say your goodbyes, you take the train back to the previous arena, and from there to where your ship is docked at.`

Reviewer 2: ```suggestion
			`	The ring platforms head back to the building, and you and Pairri say your goodbyes. You take the train back to the previous arena and from there to where your ship is docked at.`
```
(Personal preference.) This ending is rather abrupt, especially considering how long this mission is. You might want to consider adding a sentence or two (maybe some dialogue for Pairri?) to make this feel like more of a conclusion.

Author: added some stuff

	on offer
		set "license: Coalition"
		fail

> mission "Coalition Games 1"

Reviewer 3: If you condense all this into a single mission, you can just remove the "1."
```suggestion
mission "Coalition Games"
```

	name "Coalition Athletes"

> 	description "Transport three athletes to <destination> where they will compete in the Coalition games."

Reviewer 3: ```suggestion
	description "Transport three athletes to <destination> by <date> where they will compete in the Coalition games."
```

	deadline
	passengers 3

> 	to offer
> 		"coalition jobs" >= 35
> 		month == 7
> 	source
> 		near "Homeward" 3 12
> 		government "Coalition"
> 	destination "Far Home"

Reviewer 3: Just some style organizing.
```suggestion
	source
		near "Homeward" 3 12
		government "Coalition"
	destination "Far Home"
	to offer
		"coalition jobs" >= 35
		month == 7
```

	on offer
		conversation
			`You're approached by a group consisting of a Saryd, Kimek, and Arach. The Saryd speaks and the small box around their neck translates after they stop.`
			`	"The astonishing sojourner, with great joy we salute."`

> 			`	You greet the group, and the Kimek continues the conversation, "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are."`

Reviewer 3: ```suggestion
			`	You greet the group, and the Kimek continues the conversation. "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are."`
```

> 			`	The Arach speaks last, asking, "To <destination>, we must go. Compete to our hardest, we will. Transport us, would you, if upon <payment>, you agreed on?"`

Reviewer 3: ```suggestion
			`	The Arach speaks last, asking, "To <destination> by <day>, we must go. Compete to our hardest, we will. Transport us, would you, if upon <payment>, you agreed on?"`
```

			choice
				`	"That sounds good. I'll bring you there."`

> 					goto bbbb

Reviewer 3: You could probably use "end" or "accept" for this label name.

				`	"Sorry, I'm not looking for any transport jobs right now."`
					decline
				`	"Coalition games?"`
			`	"To determine the athlete with the most merit, a competition it is," the Saryd says.`
			`	"A candidate for the rank of Heliarch, one becomes, if one of the victors, one is," the Kimek adds.`
			`	"Trained hard, we have. To prove our worth in the games, we wish," the Arach ends, and they once again look at you as if expecting an answer to their original question.`
			choice
				`	"Sounds like you're all really excited about this. Alright, I'll bring you there."`
					goto bbbb
				`	"Well I wish you luck then, but sadly I'm not looking for any transport jobs right now."`
					decline
			label bbbb
			`	"Most grateful, we are," the Saryd says.`
			`	As you help them bring their luggage into your ship, they introduce themselves: the Saryd is called Ulmodi, the Kimek is Pairri, and the Arach is Tumub. They settle in their bunks to rest for the upcoming event, though as soon as Pairri closes her bunk's door you hear some slight noises. It sounds like she's doing push-ups - or maybe squats; you can't really tell how a Kimek would name such exercises.`

> 			`	Ulmodi explained that they need to get to <destination> (by <date>). It shouldn't be a problem considering you even have a Jump Drive to get there faster.`

Reviewer 3: The destination was already mentioned, and I'd suggest simply mentioning the deadline when the destination is first mentioned, so mentioning the destination again seems redundant. And as Wizrad said, we can't assume that the player has a jump drive. So perhaps either delete this line entirely, or just rework the first sentence and require that the player have a JD in order for this to offer.

Author: did some general tweaks to it

				accept
	on complete
		"coalition jobs" ++
		payment 300000

> 
> 
> 
> mission "Coalition Games 2"
> 	invisible
> 	landing
> 	to offer
> 		has "Spoder Bootle Centoro Athletes: done"
> 	source "Far Home"
> 	on offer

Reviewer 3: Since this conversation always declines, you can simply have it be the on complete conversation for the previous mission.
```suggestion
```
(I'm just going to ignore that offer condition.)

		conversation
			`As the athletes leave your ship and hand you your pay of 300,000 credits, Pairri stretches her legs, trying to get eye-level with you.`
			`	You crouch, and she says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."`

> 			`	She extends one of her legs to you, holding a small, coin-like piece. It has a symbol like the old Olympic Games on Earth, except with three interlocked rings instead of five.`

Reviewer 3: With everything in a single mission, you could use the text replacement here. I'd also condense these lines.
```suggestion
			`As the athletes leave your ship and hand you your pay of <payment>, Pairri stretches her legs, trying to get eye-level with you. You crouch, and she says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is." She extends one of her legs to you, holding a small, coin-like piece. It has a symbol like the old Olympic Games on Earth, except with three interlocked rings instead of five.`

			`	"Very long and tiring, for a first timer the games can be. Reading an entire novel in an hour, it is like," Tumub explains. "Understand, we would, if too busy to watch them, you are."`
			`	"Still, much joy, it would bring us, if to watch the games, you chose," says Ulmodi. "Rare, such a chance is, to show an outsider the wonders of our games."`
			choice
				`	"Thank you. I would be glad to view the games."`
				`	"I'm honored, but sadly I have other matters to attend to right now."`
					decline

> 			`	You take the 'ticket', and they start explaining how the games work.`

Reviewer 3: ```suggestion
			`	You take the 'ticket,' and they start explaining how the games work.`
```

			`	"Separate competitions, each species has," Tumub says. "To value each one's history, it is so."`
			`	"A grand, cooperative match, at the end of it all takes place," Ulmodi further details. "Teams consisting of all three species, formed they are. A joyous, unmatched experience, the last spectacle is."`

> 			`	Since they need to leave to prepare for the games, they can't tell you about each sport but assure you that you'll like them. You find a Heliarch agent, and they quickly point you to the line where people are checking in with their own tickets. The line must be thousands long, but you barely spend ten minutes in it. The Heliarchs taking care of admissions are incredibly efficient and keep things organized.`

Reviewer 3: ```suggestion
			`	Since they need to leave to prepare for the games, they can't tell you about each sport, but assure you that you'll like them. You find a Heliarch agent who quickly points you to the line where people are checking in with their own tickets. The line must be thousands long, but you barely spend ten minutes in it. The Heliarchs taking care of admissions are incredibly efficient and keep things organized.`
```

			`	After that, you find yet another Heliarch who shows you where you should go for each competition. Three massive transports, similar in design to old-school bullet trains, each take you to a different 'arena' for a different type of game.`

> 			`	They also explain that you'll take a separate train to go to the 'grand spectacle' at the end of it all, which will be available for boarding in any one of the three arenas after those matches are done. You thank them for the explanation and decide on which train to take.`

Reviewer 3: ```suggestion
			`	They also explain that you'll take a separate train to go to the 'grand spectacle' at the end of the Games, which will be available for boarding in any one of the three arenas after those matches are done. You thank them for the explanation and decide on which train to take.`
```

					goto wp
			
			label "seen only arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen saryd, kimek"
			choice
				`	(Take the train to the Arachi game.)`
					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen saryd, arachi"
			choice
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen kimek, arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen all"
			choice
				`	(Take the last train.)`
					goto wp
				
			
			label saryd
			apply
				set "coalition games: saryd"
			`	You arrive at a massive, stadium-like structure, only it's actually shaped like a half moon. The place overlooks a huge valley. Many small drones fly over it, transmitting live footage to the dozens of cinema-sized screens for those that can't get a clear view through the glass.`

> 			`	You notice starting gates on one of the sides of the track. A few minutes pass before an alarm sounds, and the massive crowd roars. The doors open: several Saryd burst from the gates with wheeled machines, similar to chariots, attached to them. They are also carrying bows and arrows.`

Author: didn't do the announcers translation thing now because that'd need a branch and I suck at those, also didn't get any idea for what they'd say either way. it would probably be nice to add that, though.

			`	As they move deeper, tall brambles and ledges become more common among the path, and the targets appear faster and faster. A Saryd navigates one section perfectly, nailing seven different targets in a row. You notice his picture on another monitor rises several positions, putting him near the front. It seems that hitting more targets counts towards a point system.`
			`	You weren't able to count how many started, but as the race approaches its end, you're able to see roughly thirty made it through - not many. As the Saryds cross a finish line marked by a stream, the monitors switch to a ground crew interviewing the finishers.`
			`	It looks like Ulmodi finished fourth; he's up next to be interviewed. The event seems to be over, and the crowd disperses under the supervision of Heliarch agents. Instead of heading back to the train, they're moving to a different section of the oddly-shaped stadium.`

> 			`	You go with the crowd and are shown to a room with many elevators. Eventually, everyone descends to the valley below and follows a path to an actual, full-size stadium, and a massive one at that.`

Reviewer 3: This would be a good place to put a choice in order to break up these events. Perhaps a Heliarch agent asks you about how you liked the event before sending you on to the next one. Building off of my suggestion on the lines above, there's also some information that needs to be condensed down to here.

I'd organize the choice to be something like this, but you can change these choices to whatever you think fits better.
```suggestion
			`	With the event over, the crowd disperses under the supervision of Heliarch agents. The agents guide the audience to rooms with many large elevators. You board one of the elevators, which begins descending down into the valley. During the descent, a Saryd Heliarch agent approaches you and begins speaking through her translation device. "Enjoyed the event, have you human?"`
			choice
				`	"It was a very entertaining event. I can see why the crowd was excited the whole time."`
				`	"I think the event would have been more interesting had I known what exactly was going on."`
			`	The elevator finishes its descent and the doors open. The Heliarch agent nods at your response. "Hope that you enjoy this next event, I do," she says as she begins guiding the audience along a path leading to the building you could see from above. Upon entering, you find that the massive building is a circular stadium with a rectangular court of some sort in the center. Small mounds dot the stadium grounds, while six large holes are arranged evenly around the sides of the field.`
```

> 			`	As you sit among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games. You look over the stadium grounds. It's rectangular, and small mounds dot the field. Six large holes are arranged evenly on the sides of the field.`

Reviewer 3: I'd move this bit about being the first human to witness these games to the line talking about waiting, and move the description of the stadium grounds to when you first enter. (In my suggestions I have moved the second sentence to the one above this and the first sentence to the one below this.) This way we flow from entering and describing the stadium to pondering on the player's place in all this and then moving on to what's happening with the next event, instead of entering the stadium, thinking about the player's place, *then* describing the stadium, then moving on to the event.
```suggestion
```

> 			`	Half an hour passes, and a Kimek sitting next to you with a translation device explains that the contestants are resting. A couple more minutes pass before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. You see that Ulmodi is on the blue team. Several large spheres are placed at the center of the stadium in a circle, and one member from each team stands around them.`

Reviewer 3: Repeating what I suggested with the first event, I'd mention another announcer at this point. This also gives the Kimek a reason to be telling you that the contestants are resting, and while you wait we can talk about being the first human to witness these events.
```suggestion
			`	After the audience is seated, an announcer once again comes on the loudspeaker. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting, and the next event will be starting shortly. As you sit waiting among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games."
			`	Half an hour passes before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. The members of each team are introduced to the crowd and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
```

			`	The game starts. It's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience; fortunately, nets are in place to keep the spheres within the playing area.`
			`	Eventually, a sphere finds its way into a hole, which lights up with the color of the team who scored: red. Much of the crowd jumps up and screams. You're not sure if everyone is a fan of the red team or if they're cheering simply because a point was scored. The sphere is brought back up automatically via a tube close to the hole and it's back in the game.`
			`	The game continues, and each team accumulates a few points. The blue team, leading, scores a fifth point, and the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The Kimek with the translator explains what is happening.`

> 			`	"The winning score, they have reached! The winners, the blue team are!" They say, still in a cheering mood. After everyone settles down, the members of the teams are shown on the replay screen, going from the team that scored the least all the way to the blue team at the end. Ulmodi is placed third, which must mean a lot considering him and the other two above him celebrate by running laps around the inside of the stadium.`

Reviewer 3: Simple "the" to "a," as "the" implies to me that the screen was previously introduced. I'd also just add a small snippet about how Ulmodi is placed in third because of his performance in both events.
```suggestion
			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, going from the team that scored the least all the way to the blue team at the end. Between the two events, Ulmodi is placed third, which must mean a lot considering him and the other two above him celebrate by running laps around the inside of the stadium.`
```

> 			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. The crowd begins to file out.`
> 			`	You get back to where the train dropped you off and see that another one has arrived on the opposite side. A Heliarch agent explains that it is headed to the final arena where the cooperative game will take place.`

Reviewer 3: I'd simply condense these two lines.
```suggestion
			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. The crowd begins to file out. You get back to where the train dropped you off and see that another one has arrived on the opposite side. A Heliarch agent explains that it is headed to the final arena where the cooperative game will take place.`
```

				goto choice
			
			
			label kimek
			apply
				set "coalition games: kimek"
			`	The overly-packed train takes you to a savanna plain with a colossal skyscraper. It looks like a stadium has been vertically stretched tenfold. In the center are several rings held in place by beams connected to the seating area.  Hundreds of Heliarch agents direct the nonstop stream of Kimek entering to their seats in an orderly fashion.`
			`	You ask one of the agents about the multiple 'floors' and rings. They explain that because of the Kimek's disproportionately high population, their own competition sees much higher attendance than the other species. There are twenty floors, and each is big enough to seat a stadium-full.`
			`	"To other floors, at any point you may go," they say. "To the elevators, you must go, if to see another floor, you wish."`
			`	You ask them for a way to find out where Pairri will be competing; seeing as you don't really understand any of this, you should at least stick to a floor where someone you know will be participating.`

> 			`	They point you to a separate group of agents inside with computers directing the never-ending crowd to their preferred floors. Eventually, they point you to the sixteenth floor, and you head to the elevators. They're just as packed as the train was. After a crowded ride, you find a seat. Among the thousands of Kimek on this floor alone, you spot an occasional group of Arachi or a loner Saryd. Everyone eagerly waits for the match to begin.`

Author: I prefer to keep the very start as is, so used that instead of `The Kimek events take place in`

			`	Eventually, after many of these rounds and rotational swap outs, Pairri and her next opponent wait as the other forty-nine pairs finish their own fights, then move to the center of the field.`
			`	"Undefeated, both of them are," the Arach behind you says. "The floor victor, the winner of this final match will be."`
			`	The whistle sounds again, and Pairri charges at her slightly larger opponent, pressuring them toward the edge. They redirect the force by going to their side, and the fight quickly turns into a battle of attrition near the edge of the ring. Her opponent repositions themself closer to the middle and tries to charge, but she closes the gap. They continue wrestling before Pairri presses the other Kimek towards the border again. Suddenly, her opponent manages to grip her with their frontal legs, and by lifting their own body with the other four, manages to push her off the ground and into the air.`
			`	Both fall outside the ring, but Pairri hits the ground first. You hear the whistle once again, and as they get up and approach each other for what must be the Kimek equivalent of a handshake, it becomes clear that Pairri has lost.`

> 			`	Heliarch agents make sure everyone leaving the floor proceeds in an orderly fashion, and you take the elevators back down, returning to the entrance. You see a second train there, and the Arach that talked to you explains that it leads to where the cooperative game will take place.`

Reviewer 3: I'd add an extra sentence at the beginning of this, simply as a transition between the event and leaving.
```suggestion
			`	With the event finally over, the audience begins to leave. Heliarch agents make sure everyone leaving the floor proceeds in an orderly fashion, and you take the elevators back down, returning to the entrance. You see a second train there, and the Arach that talked to you explains that it leads to where the cooperative game will take place.`
```

			`	After they finish, the monitor switches to a black screen, which after a moment displays live footage of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, and Pairri makes a sound that you think sounds like an excited scream - or, at least, a Kimek version of such a thing.`
			`	"Selected, my friends have been!" She says. "Made it, they have!"`
			`	A few more words are said after the crowd has settled, and the six that were selected are invited into the Punishers. Pairri explains they are being considered for the rank of Heliarch and will go to the Ring of Friendship to continue their journey.`

> 			`	The ring platforms head back to the building, and you board the train with Pairri to head back to one of the previous areans, and from there to where your ship is docked at. During the train rides she used some device to call friends and family members, who congratulate her on getting so far. She still can't seem to contact Ulmodi or Tumub, though.`

Reviewer 2: ```suggestion
			`	The ring platforms head back to the building, and you board the train with Pairri to head back to one of the previous arenas, and from there to where your ship is docked at. During the train rides she used some device to call friends and family members, who congratulate her on getting so far. She can't seem to contact Ulmodi or Tumub, though.`
```

			`	"Busy with the Heliarchs they must be." She guesses. "That there with them I am not, a shame it is."`
			choice

> 				`	"So are they themselves Heliarchs now?"`

Reviewer 2: ```suggestion
				`	"So are they Heliarchs now?"`
```

				`	"You almost won this time, so I'm sure you'll make it next time."`
					goto nexttime
			`	"Not yet. A few more short trials they must pass, before elevated to that rank they are." She explains. "Similar to the standard process, it is. Only, for those selected during the games, much faster said process is."`
				goto ending
			label nexttime
			`	"Thanks Captain. Even if next year watching my matches you are not, do my best I promise I will. Better than this last time I will do."`
			label ending

> 			`	A dozen Kimek suddenly come running towards you two, Pairri telling you it's her family. She tries to introduce you to all of them, but they seem much more interested in telling her how proud they were to see her compete. After they calm down a bit, you say goodbye to Pairri, wishing her good luck for next year's games, and head back to your ship.`

Reviewer 2: ```suggestion
			`	A dozen Kimek suddenly come running towards you two, and Pairri tells you it's her family. She tries to introduce you to all of them, but they seem much more interested in telling her how proud they were to see her compete than they are in meeting you. After they calm down a bit, you say goodbye to Pairri, wish her good luck for next year's games, and head back to your ship.`
```
